### DISCUSSION TITLE
[general] How about adding an option to turn on printing object loading/saving time when running workflow?

### DISCUSSION THREAD

**Discussion by psychelzh** [ropensci/targets#1337] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I found that sometimes the object loading/saving time is quite an overhead, and thus printing such information could be of great help to diagnose the bottleneck and change my script accordingly.

**Comment by wlandau** [comment]
Added the storage size of each target to print messages for verbose reporters: 2aba17aba7b0389f9c8642c07adbe36f0c4af5b2. Storage size + execution time are orthogonal metrics that together should help troubleshoot.

```r
> tar_make()
▶ dispatched target index
● completed target index [0 seconds, 97 bytes]
▶ dispatched branch a_ef1ae9b9954fc770
● completed branch a_ef1ae9b9954fc770 [0 seconds, 47 bytes]
▶ dispatched branch a_6d03443cebc2cf01
● completed branch a_6d03443cebc2cf01 [0 seconds, 48 bytes]
● completed pattern a 
▶ ended pipeline [0.907 seconds]
```
